shared/scripts/ci-tests.sh

```diff
@@ -66,7 +66,8 @@ setup() {
 }
 
 run_tests() {
-  azk nvm gulp test --slow
+  unset TEST_GREP
```

Comment:
What is the motivation for this cleaning? It conflicts with some internal thing gulp?